lib/phoenix/endpoint.ex

```diff
@@ -1060,11 +942,11 @@ defmodule Phoenix.Endpoint do
       by `Phoenix.Token`. By default tokens are valid for 2 weeks
 
   """
-  defmacro socket(path, module, opts \\ []) do
-    module = Macro.expand(module, %{__CALLER__ | function: {:socket_dispatch, 2}})
+  defmacro socket(_path, _module, _opts \\ []) do
```

Comment:
We most likely want to still support the old way as usual

Reply:
Yeah, I removed stuff in here mostly to make sure I catch everything. Making this propely backwards compatibility preserving will make sense once other questions are dealt with.